android/modules/ui/src/java/ti/modules/titanium/ui/widget/listview/TiListView.java

```diff
@@ -895,7 +895,10 @@ public View layoutHeaderOrFooterView(TiViewProxy viewProxy)
 
 	protected void processSections(Object[] sections)
 	{
-
+		// Release current sections in the lest
+		for (ListSectionProxy listSectionProxy : this.sections) {
+			listSectionProxy.release();
```

Comment:
Maybe this should be `releaseViews()` ?

If we release the whole proxy then we can't re-add it again later